documentation/ide.asciidoc

```diff
@@ -58,4 +58,4 @@ We hope you love `devonfw-ide`. However, if you don't and want to get rid of it
 devon ide uninstall
 ```
 This will remove `devonfw-ide` from all central places of your OS (user home directory such as `scripts`, `.devon`, `.bashrc`, as well as windows registry, etc.).
-However, it will not remove your current installations (or shared link:software.asciidoc[software] folder). So after running this `uninstall`, simply remove your `DEVON_IDE_HOME` directory of all `devonfw-ide` installations and potential shared link:software.asciidoc[software] folder. You may also want to clean up your `~/Downloads` directory from files downloaded by `devonfw-ide`. We do not automate this as deleting a directory is a very simple manual step and we do not want to take responsibility for severe data loss if your workspaces contained valuable work.
+However, it will not remove your current installations (or shared link:software.asciidoc[software] folder). So after running this `uninstall`, simply remove your `DEVON_IDE_HOME` directory of all `devonfw-ide` installations and potential shared link:software.asciidoc[software] folder. You may also want to clean up your `~/Downloads` directory from files downloaded by `devonfw-ide`. We do not automate this as deleting a directory is a very simple manual step and we do not want to take responsibility for severe data loss if your workspaces contained valuable work.
```

Comment:
I can not see any reasonable change here. Do you have your EOL-Style configured properly for git or how can we avoid such phantom changes that spam the diffs?